app/models/service_template.rb

```diff
@@ -1,4 +1,7 @@
 class ServiceTemplate < ApplicationRecord
+  include ReservedMixin
+  reserve_attribute :internal, :boolean
```

Comment:
These lines should be moved down by the other `include` statements around line 38.